frontend/src/components/Header/Header.js

```diff
@@ -0,0 +1,117 @@
+import { React, useState } from 'react';
+import './Header.scss';
```

Comment:
is there any particular reason why you are not using css modules?

https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

Reply:
This was how the worksheet told us to do it, I believe. I can change it thought if thats the preferred way